--- slurm-21.08.6_bak/src/plugins/openapi/v0.0.37/jobs.c        2022-02-24 20:10:29.000000000 +0100
+++ slurm-21.08.6/src/plugins/openapi/v0.0.37/jobs.c    2022-07-14 15:02:59.492972019 +0200
@@ -773,9 +773,7 @@
                                                (i /
                                                 j->cores_per_socket[sock_inx]));
                                        data_t *core = data_key_set_int(
-                                               cores,
-                                               (i %
-                                                j->cores_per_socket[sock_inx]));
+                                               cores, i);
 
                                        if (bit_test(j->core_bitmap_used,
                                                     bit_inx)) {
